EngineTest/Unit/Organization/OrganizationTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FlagsmithEngine.Organization.Models;
+using Xunit;
+namespace EngineTest.Unit.Organization
+{
+    public class OrganizationTest
```

Comment:
Can we use english spellings please as per the python client? `Organisation`, not `Organization`.